akka-projection-testkit/src/main/scala/akka/projection/testkit/ProjectionTestKit.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection.testkit
+
+import akka.Done
+import akka.actor.testkit.typed.TestKitSettings
+import akka.actor.testkit.typed.scaladsl.{ ActorTestKit, _ }
+import akka.annotation.ApiMayChange
+import akka.projection.Projection
+
+import scala.concurrent.duration._
+import scala.concurrent.{ Future, Promise }
+@ApiMayChange
+object ProjectionTestKit {
+  def apply(testKit: ActorTestKit): ProjectionTestKit =
+    new ProjectionTestKit(testKit)
+}
+
+@ApiMayChange
+final class ProjectionTestKit private[akka] (testKit: ActorTestKit) {
```

Comment:
Like with the new PersistenceTestKit, the ProjectionTestKit gets a ActorTestKit (typed)